CONTRIBUTING.rst

```diff
@@ -88,6 +88,8 @@ without using a local copy.  This can be convenient for small fixes.
 
     The built documentation should be available in ``doc/en/_build/html``,
     where 'en' refers to the documentation language.
+    You can launch a simple local server by running ``bash doc/en/start_doc_server.sh``
```

Comment:
oh uh, I'm not sure we need a server at all -- usually you can just `firefox doc/en/index.html` or whatever -- we should proably delete `start_doc_server.sh` (especially because it is not portable to windows)

Reply:
Indeed, do you mind removing the local server and removing this docs @xinbinhuang ?

Reply:
I generally launch a simple server because I believe it provides better navigation than the `file://`.   But probably not needed for sphinx built documentation. We can add it back if really need to.

I just removed the `start_doc_server.sh`. PTAL :)